.github/ISSUE_TEMPLATE/QUESTION.md

```diff
@@ -1,14 +1,8 @@
 <!--
 You can have your question reach the wider Sentry community via:
 
-StackOverflow.com with the Sentry tag:
-https://stackoverflow.com/questions/ask
-
-Sentry forum is also a great knowledge base:
-https://forum.sentry.io/
-
 Chat online with other Sentry users:
-https://discord.gg/Ww9hbqr
+https://discord.gg/sentry
```

Comment:
Huh, I honestly don't think that this issue template gets shown anywhere. Maybe we should just delete it? I can make a separate PR for that, since it's out of scope in this PR

Reply:
#2472